tests/cases/fourslash/completionImportMeta.ts

```diff
@@ -1,5 +1,24 @@
 /// <reference path='fourslash.ts' />
 
-////import./**/
+// @Filename: a.ts
+////import./*1*/
 
-verify.completions({ marker: "", exact: "meta" });
+// @Filename: b.ts

```

Comment:
As part of this PR I would like you to add another test case file where global doesn't have meta and you look for completions at `import.`, `import.meta.` and `import./*here*/meta` in both these test case files (the one currently here and one without global import meta declaration)

Reply:
Let me know if the most recent change is what you requested!